helm/cluster-autoscaler-app/templates/service.yaml

```diff
@@ -4,13 +4,20 @@ metadata:
   name: {{ .Values.name }}
   namespace: {{ .Values.namespace }}
   labels:
-    giantswarm.io/service-type: "{{ .Values.serviceType }}"
-    app: {{ .Values.name }}
+    {{- include "labels.common" . | nindent 4 }}
+    # kubernetes.io/cluster-service are kept to be displayed in kubectl cluster-info
     kubernetes.io/cluster-service: "true"
     kubernetes.io/name: {{ .Values.name }}
+    giantswarm.io/monitoring: "true"
```

Comment:
Shouldn't this be annotation?

Reply:
We decided to go with a label as explained: https://github.com/giantswarm/giantswarm/blob/12511-tc-prom-spec/content/docs/architecture-specs-adrs/specs/configuration-of-targets-in-tc-prometheus.md#labels-and-annotations

As this is not merged yet, we did not communicate this

Reply:
Sounds like there's some misalignment https://intranet.giantswarm.io/docs/monitoring/monitoring_of_apps/

//cc @JosephSalisbury @piontec

Reply:
The discussion was brought up https://github.com/giantswarm/giantswarm/pull/12512/files#r465583355